src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+        Task[] taskList = new Task[100];
+        handleUserInput(taskList);
+        printExitMessage();
+    }
+
+    public static void handleUserInput(Task[] taskList) {
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine().trim();
+
+        while (!userInput.equals("bye")){
+            if (userInput.equals("list")) {
```

Comment:
Try to be consistent in the while loop and if loop, as some got space before the open bracket, some no

Reply:
Noted. Thanks